src/pages/links.tsx

```diff
@@ -0,0 +1,46 @@
+import Head from 'next/head'
+
+import Portal from '@/components/links/portal'
+import Footer from '@/components/sections/footer'
+import { ILinkRecord } from '@/types/links'
+
+interface LinkProps {
+  links: ILinkRecord[]
+}
+
+const Links = ({ links }: LinkProps) => {
+  return (
+    <>
+      <Head>
+        <title>Links from Thinc.</title>
+      </Head>
+      <Portal links={links} />
+      <Footer />
+    </>
+  )
+}
+
+export async function getStaticProps() {
+  const requestHeaders: HeadersInit = new Headers()
+  requestHeaders.set('Authorization', `Bearer ${process.env.AIRTABLE_API_KEY}`)
+  requestHeaders.set('X-Airtable-Client-Secret', process.env.AIRTABLE_CLIENT_SECRET ?? '')
+
+  const res = await fetch('https://api.airtable.com/v0/appTJRDfmc9EBSDvO/Links', {
+    headers: requestHeaders,
+  })
+
+  const linkRecords: { href: string; Title: string }[] = (await res.json()).records.map(
```

Comment:
I think we can use `ILinkRecord` here instead of `{ href: string; Title: string }`, just need to rename `Title` -> `title`